Tests/KituraSampleRouterTests/KituraTest.swift

```diff
@@ -41,18 +41,22 @@ extension KituraTest {
 
     func performServerTest(asyncTasks: @escaping (XCTestExpectation) -> Void...) {
         let router = RouterCreator.create()
-        Kitura.addHTTPServer(onPort: 8090, with: router)
-        Kitura.start()
-        sleep(1)
```

Comment:
Added an issue to remove using `started` - https://github.com/IBM-Swift/Kitura/issues/865